Live/Classes/Tree.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#include "ResourceObstacle.h"
+#include "Fruit.h"
+
+#include <queue>
+
+#define AVERAGE_DAY 1
+
+#define TREE_SCALE_FACTOR 0.5
+class Tree : public ResourceObstacle {
+public:
+    Tree(const std::string& sprite_sheet, float pos_x, float pos_y);
+    virtual void update();
+    virtual Item* gather(cocos2d::Point, Direction);		// Gather a resource from an obstacle you are facing
+    virtual ~Tree();
+
+private:
+    std::queue<Cherry*> m_fruits;
+    // store what the last day a fruit blossomed
```

Comment:
nit: store the last day a fruit blossomed

Reply:
already am doing that? it doesn't have to be attached to each fruit though, since we only need to know when the last fruit blossomed, in order that we can take that into consideration when deciding when to generate a new fruit.

Reply:
I mean your comment grammar.